devkit/Makefile

```diff
@@ -21,12 +21,13 @@ INSTALL_DIR := /usr/local/bin
 # version variables
 #
 # tag and repo values may be overriden from the outside
+REGISTRY ?= asdrepo.isus.emc.com:8085
```

Comment:
please remove this line.

Reply:
OK, this line is removed, but I notice in **_tests/ci/sh_**, we have similar information, do we need to remove it?

Reply:
for old code, just keep it.  just make sure new code does not introduce internal registry.